apps/meteor/tests/e2e/07-emoji.spec.ts

```diff
@@ -104,7 +102,7 @@ test.describe('[Emoji]', function () {
 
 			test('expect send the emoji', async () => {
 				await mainContent.addTextToInput(' ');
-				await mainContent.getPage().keyboard.press('Enter');
+				await mainContent.keyPress('Enter');
```

Comment:
keyboard keys could be in an enum

Reply:
this pr is already to big xd, I'll note this